modules/serverless-tls-ensuring-seamless-ca-rotation-eventing.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * /serverless/Eventing/serverless-config-tls-encryption-eventing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-tls-ensuring-seamless-ca-rotation-eventing_{context}"]
+= Ensuring seamless CA rotation
+
+Ensuring seamless CA rotation is essential to avoid service downtime or handle emergencies.
+
+.Prerequisites
+
+* You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
+* You have installed the {ServerlessOperatorName}.
+* You have installed the {cert-manager-operator}.
+* You have installed the OpenShift (`oc`) CLI.
+
+.Procedure
+
+. Create a new CA certificate.
+. Add the public key of the new CA certificate to the CA trust bundles. 
++
+Make sure you also keep the public key of the existing CA.
+. Ensure all clients use the latest CA trust bundles.
++
+Knative Eventing components will automatically reload the updated CA trust bundles. For custom workloads consuming trust bundles, reload or restart them as needed.
+. Update the `knative-eventing-ca-issuer` `ClusterIssuer` to reference the secret containing the CA certificate created in step 1.
+. Force `cert-manager` to renew certificates in the `knative-eventing namespace`.
++
+For more information about `cert-manager`, see link:https://cert-manager.io/docs/usage/certificate/#reissuance-triggered-by-user-actions[Reissuance triggered by user actions].
+. As soon as the CA rotation is fully completed, remove the public key of the old CA from the trust bundle `ConfigMap`.
```

Comment:
```suggestion
. As soon as the CA rotation is fully completed, remove the public key of the old CA from the trust bundle config map.
```
Use the `ConfigMap` spelling only if you are referring to the actual `ConfigMap` object. For general references to config maps, use the spelling of "config map".